fuel-forge/src/test/kotlin/com/github/babedev/fuel/forge/FuelForgeTest.kt

```diff
@@ -0,0 +1,116 @@
+                    Assert.assertThat(result.component1(), CoreMatchers.notNullValue())
+                    Assert.assertThat(result.component2(), CoreMatchers.instanceOf(Result.Failure::class.java))
+                }
+    }
+
+    @Test
+    fun forgeTestResponseDeserializerObject() {
+        Fuel.get("/user-agent")
+                .responseObject(httpBinUserDeserializer) { _, _, result ->
+                    Assert.assertThat(result.component1(), CoreMatchers.notNullValue())
+                    Assert.assertThat(result.component2(), CoreMatchers.notNullValue())
+                }
+    }
+
+    @Test
+    fun forgeTestResponseDeserializerObjectError() {
+        Fuel.get("/useragent")
+                .responseObject(httpBinUserDeserializer) { _, _, result ->
+                    Assert.assertThat(result.component1(), CoreMatchers.notNullValue())
+                    Assert.assertThat(result.component2(), CoreMatchers.instanceOf(Result.Failure::class.java))
+                }
+    }
+
+    @Test
+    fun forgeTestResponseHandlerObject() {
+        Fuel.get("/user-agent")
+                .responseObject<HttpBinUserAgentModel>(httpBinUserDeserializer, object : Handler<HttpBinUserAgentModel> {
+                    override fun success(request: Request, response: Response, value: HttpBinUserAgentModel) {
+                        Assert.assertThat(value, CoreMatchers.notNullValue())
+                    }
+
+                    override fun failure(request: Request, response: Response, error: FuelError) {
+                        Assert.assertThat(error, CoreMatchers.notNullValue())
+                    }
+
+                })
+    }
+
+    @Test
+    fun forgeTestResponseHandlerObjectError() {
+        Fuel.get("/useragent")
+                .responseObject(httpBinUserDeserializer, object : Handler<HttpBinUserAgentModel> {
+                    override fun success(request: Request, response: Response, value: HttpBinUserAgentModel) {
+                        Assert.assertThat(value, CoreMatchers.notNullValue())
+                    }
+
+                    override fun failure(request: Request, response: Response, error: FuelError) {
+                        Assert.assertThat(error, CoreMatchers.instanceOf(Result.Failure::class.java))
+                    }
+
+                })
+    }
+
+    data class IssueInfo(val id: Int, val title: String, val number: Int)
+
+    private val issueInfoDeserializer = { json: JSON ->
+        ::IssueInfo.create.
+                map(json at "id").
+                apply(json at "title").
+                apply(json at "number")
+    }
+
+    @Test
+    fun testProcessingGenericList() {
+        Fuel.get("https://api.github.com/repos/kittinunf/Fuel/issues").responseObjects(issueInfoDeserializer) { _, _, result ->
+            val issues = result.get()
+            Assert.assertNotEquals(issues.size, 0)
+            Assert.assertThat(issues[0], CoreMatchers.isA(IssueInfo::class.java))
+        }
+    }
+}
```

Comment:
Can you remove the new line at the end of the file?

Reply:
Fixed 😎